redis/src/cluster.rs

```diff
@@ -970,8 +970,8 @@ pub(crate) fn get_connection_info(
             password: cluster_params.password,
             username: cluster_params.username,
             client_name: cluster_params.client_name,
-            use_resp3: cluster_params.use_resp3,
-            ..Default::default()
+            protocol: cluster_params.protocol,
+            db: 0,
```

Comment:
why do you remove the ..Default::default() here, and added db: 0 instead, and in other places (e.g. [sentinel.rs](https://github.com/amazon-contributing/redis-rs/pull/84/files#diff-6e43330d9f65fae075a9bb65ccb71db69f54fe2da7a94512ec00c7f423bf1058)) you remove the db and use the defaults?

Reply:
internally, I prefer to remove the default, because it hides fields that are added and should be configured. In external documentation I give the simplified usage.